external-crates/move/move-compiler/src/diagnostics/codes.rs

```diff
@@ -232,6 +232,8 @@ codes!(
         DeadCode: { msg: "dead or unreachable code", severity: Warning },
         StructTypeParam: { msg: "unused struct type parameter", severity: Warning },
         Attribute: { msg: "unused attribute", severity: Warning },
+        StructType: { msg: "unused struct type", severity: Warning },
+        Function: { msg: "unused (private) function", severity: Warning },
```

Comment:
I'd probably just stick with `"unused function"`, since we can re-use this warning with public(friend) eventually